test/test_check_branch.py

```diff
@@ -70,3 +70,18 @@ def expect_exception(branch, deployment, message):
                              'prod',
                              "Branch 'feature/foo' cannot be deployed to 'prod', "
                              "only one of {'sandbox'} or personal deployments.")
+
+        supported_vars = ['CI_COMMIT_REF_NAME', 'GITHUB_BASE_REF', 'GITHUB_HEAD_REF']
+
+        for active_env_var in supported_vars:
+            if active_env_var == 'GITHUB_HEAD_REF':
+                base_branch = ['foo/bar']
+            else:
+                base_branch = ['develop', 'prod']
+            for branch in base_branch:
+                with self.subTest(branch_name=branch, active_env_variable=active_env_var):
```

Comment:
```suggestion
                with self.subTest(branch_name=branch, active_env_variable=active_env_var):
```

Keys should be consistent with variable names.